src/mlpack/methods/kernel_pca/kernel_rules/naive_method.hpp

```diff
@@ -77,7 +77,9 @@ class NaiveKernelRule
 
   // Swap the eigenvalues since they are ordered backwards (we need largest to
   // smallest).
-  for (size_t i = 0; i < floor(eigval.n_elem / 2.0); ++i)
+  omp_size_t num_eigenvals = floor(eigval.n_elem / 2.0);
```

Comment:
Please use camel casing for all names, see https://github.com/mlpack/mlpack/wiki/DesignGuidelines#naming-conventions for details.